src/lib/utils/track.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from "react"
-import { track as _track, Track as _Track } from "react-tracking"
+import _track, { Track as _Track } from "react-tracking"
```

Comment:
Not even sure why this worked before, the `_track` function is a default export, not a named one.

Reply:
wonder if babel is doing some magic threre

Reply:
Perhaps yeah.